latex/content/debug.tex

```diff
@@ -1,6 +1,6 @@
 \subsection{Debug}
 
-\begin{frame}[fragile]{\lstinline+\\overfullrule+}
+\begin{frame}[fragile]{\texttt{\textbackslash overfullrule}}
```

Comment:
Put color change in beamerstyle as suggested by mnoethe.
Set color to vertex light grey, to indicate typewriter with good contrast.